alvr/client_openxr/src/interaction.rs

```diff
@@ -168,11 +168,12 @@ pub struct InteractionContext {
     xr_session: xr::Session<xr::OpenGlEs>,
     xr_system: xr::SystemId,
     extra_extensions: Vec<String>,
-    platform: Platform,
+    pub platform: Platform,
     pub action_set: xr::ActionSet,
     pub button_actions: HashMap<u64, ButtonAction>,
     pub hands_interaction: [HandInteraction; 2],
     multimodal_handle: Option<MultimodalMeta>,
+    pub relative_head_orientation: Quat,
```

Comment:
This should probably need a comment, saying that this is the relative head orientation since the refernce space first initialization. ALso it might be good to mention why it's needed (workaround for Pico eye tracking)